components/automate-gateway/handler/iam/v2beta/tokens/tokens.go

```diff
@@ -67,10 +74,18 @@ func (s *Server) GetToken(
 // UpdateToken uses a token ID to update the Name/Active fields only
 func (s *Server) UpdateToken(
 	ctx context.Context, req *pb_req.UpdateTokenReq) (*pb_resp.UpdateTokenResp, error) {
+	// If active value not specified, default to True.
```

Comment:
Or should we leave the existing value alone if not specified? Not sure which is preferable.

Reply:
The previous behavior was that it was defaulting to `false` if the request did not contain the `active` field, which we almost certainly don't want. We should either default it to `true` like create, allow partial updates (which none of our other APIs do and would require changes to the domain), or return a 400 if they didn't pass the active field on update, but that's much more breaking change. Given all of that, seemed like defaulting to true was the best option!